lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -259,22 +353,19 @@ class MockTimers {
     this.#isEnabled = activate;
   }
 
-  tick(time = 1) {
-    if (!this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'You should enable MockTimers first by calling the .enable function',
-      );
-    }
+  #isValidDateWithGetTime(maybeDate) {
+    return (
+      maybeDate instanceof globalThis.Date &&
```

Comment:
instanceof can be faked or broken, and doesn't work cross-realm - the only reliable way to check this is https://github.com/inspect-js/is-date-object/blob/main/index.js#L3-L11

Reply:
Noted, will try to solve this way